src/lib/components/Rating/RatingContext.tsx

```diff
@@ -1,9 +1,12 @@
 import { createContext, useContext } from 'react';
+import type { FlowbiteSizes } from '../Flowbite/FlowbiteTheme';
 
-export type Size = 'sm' | 'md' | 'lg';
+export interface StarSizes extends Pick<FlowbiteSizes, 'sm' | 'md' | 'lg'> {
```

Comment:
IMO this should be next to RatingProps interface in index.tsx but your call

Reply:
Looks like a good idea.